code/datums/supplypacks/costumes_vr.dm

```diff
@@ -0,0 +1,269 @@
+	containertype = /obj/structure/closet/crate
+	containername = "Varsity jackets crate"
+
+/datum/supply_packs/randomised/costumes/leathergear_vr
+	name = "Leather gear"
+	num_contained = 5
+	contains = list(
+			/obj/item/clothing/suit/leathercoat,
+			/obj/item/clothing/suit/storage/leather_jacket_alt,
+			/obj/item/clothing/suit/storage/toggle/leather_jacket,
+			/obj/item/clothing/suit/storage/toggle/leather_jacket/sleeveless,
+			/obj/item/clothing/suit/storage/toggle/leather_jacket/nanotrasen,
+			/obj/item/clothing/suit/storage/toggle/leather_jacket/nanotrasen/sleeveless,
+			/obj/item/clothing/under/pants/chaps,
+			/obj/item/clothing/under/pants/chaps/black,
+			/obj/item/clothing/under/harness,
+			/obj/item/clothing/shoes/leather,
+			/obj/item/clothing/shoes/boots/jungle,
+			/obj/item/clothing/shoes/boots/jackboots,
+			/obj/item/clothing/shoes/boots/cowboy,
+			/obj/item/clothing/shoes/boots/cowboy/classic,
+			/obj/item/clothing/shoes/boots/cowboy/snakeskin,
+			/obj/item/clothing/accessory/chaps,
+			/obj/item/clothing/accessory/chaps/black,
+			/obj/item/clothing/accessory/collar/spike,
+			/obj/item/clothing/gloves/fingerless,
+			/obj/item/clothing/gloves/botanic_leather,
+			/obj/item/clothing/head/cowboy_hat,
+			/obj/item/clothing/head/cowboy_hat/black
+			)
+	cost = 50
+	containertype = /obj/structure/closet/crate
+	containername = "Leather gear crate"
+
+/datum/supply_packs/costumes/eyewear_vr
+	name = "Eyewear crate"
+	contains = list(
+			/obj/item/clothing/glasses/eyepatch,
+			/obj/item/clothing/glasses/fakesunglasses,
+			/obj/item/clothing/glasses/fakesunglasses/aviator,
+			/obj/item/clothing/glasses/fluff/science_proper,
+			/obj/item/clothing/glasses/fluff/spiffygogs,
+			/obj/item/clothing/glasses/gglasses,
+			/obj/item/clothing/glasses/monocle,
+			/obj/item/clothing/glasses/regular,
+			/obj/item/clothing/glasses/regular/hipster,
+			/obj/item/clothing/glasses/regular/scanners,
+			/obj/item/clothing/glasses/threedglasses
+			)
+	cost = 50
+	containertype = /obj/structure/closet/crate
+	containername = "Eyewear crate"
+
+/datum/supply_packs/randomised/costumes/gloves_vr
+	name = "Gloves crate"
+	num_contained = 4
+	contains = list(
+			/obj/item/clothing/gloves/black,
+			/obj/item/clothing/gloves/blue,
+			/obj/item/clothing/gloves/botanic_leather,
+			/obj/item/clothing/gloves/brown,
+			/obj/item/clothing/gloves/evening,
+			/obj/item/clothing/gloves/fingerless,
+			/obj/item/clothing/gloves/yellow,
+			/obj/item/clothing/gloves/fyellow,
+			/obj/item/clothing/gloves/green,
+			/obj/item/clothing/gloves/grey,
+			/obj/item/clothing/gloves/light_brown,
+			/obj/item/clothing/gloves/orange,
+			/obj/item/clothing/gloves/purple,
+			/obj/item/clothing/gloves/rainbow,
+			/obj/item/clothing/gloves/red,
+			/obj/item/clothing/gloves/white,
+			/obj/item/clothing/gloves/sterile/latex,
+			/obj/item/clothing/gloves/sterile/nitrile,
+			/obj/item/clothing/gloves/swat,
+			/obj/item/clothing/gloves/tactical
```

Comment:
SWAT and tactical gloves are armored. Tactical are even insulated. Remove these.